Final Project/Editor Interface/editor_interface_final.py

```diff
@@ -1,24 +1,21 @@
 #!/usr/bin/python
-# Default FOV is 10
+# Default FOV is 10 
 
 import matplotlib
-import matplotlib.pyplot as plt
-import mpld3
-import pandas as pd
+import matplotlib.pyplot as plt, mpld3 
+import pandas as pd 
 from mpld3 import plugins
 from mpld3 import utils
 from matplotlib.patches import Circle
 from svgpath2mpl import parse_path
 import matplotlib.lines as mlines
 from IPython.core.display import display, HTML
-import math
 display(HTML("<style>.container { width:100% !important; }</style>"))
 
-## The databases are imported first
-ng = pd.read_csv('Database/NGC.csv')
-ms = pd.read_csv('Database/messier_objects.csv')
-cb = pd.read_csv('Database/constellation_borders.csv')
-ty = pd.read_csv('Database/tycho-1.csv', nrows=5031)
+ng = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/NGC.csv')
+ms = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/messier_objects.csv')
+cb = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/constellation_borders.csv')
+ty = pd.read_csv('/Users/yashakaushal/Documents/summer_project/interface/tycho_1.csv')
 
```

Comment:
So, if we use these database importing code, it won't work in someone else local machines. The editor would have change these manually. Can we keep these as what it is right now? It'll work in generally anyone's computer.

Reply:
Right. If we keep all these .csv files in a folder in the repo, we can only mention that path and the user while cloning need not change anything.

Reply:
Fixed this by defaulting the file path to Database folder.